rules/coding-style/tests/Rector/ClassMethod/UnSpreadOperatorRector/Fixture/handle_multiple_items_to_array.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\ClassMethod\UnSpreadOperatorRector\Fixture;
+
+final class HandleMultipleItemsToArray
+{
+    public function go()
+    {
+        $this->run('John', 1, 2, 3, 4, 5);
+    }
+
+    public function run(string $name, ...$items)
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\ClassMethod\UnSpreadOperatorRector\Fixture;
+
+final class HandleMultipleItemsToArray
+{
+    public function go()
+    {
+        $this->run('John', [1, 2, 3, 4, 5]);
```

Comment:
@samsonasik Hey, just FYI, I had to update this rules to group items to an array like this.
It broke some code on Symplify.

Reply:
Thank you